### PR TITLE
chore: remove usage of modelsDir option

### PR DESCRIPTION
**Pull request by arnaud-moncel**
## Pull Request checklist:

- [X] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Test manually the implemented changes
- [X] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: src/services/config-store.js [hunk 1]
```diff
@@ -58,6 +54,10 @@ class ConfigStore {
     if (options.onlyCrudModule) {
       this.logger.warn('The use of onlyCrudModule is deprecated. Please remove the option in Forest initialization.');
     }
+
+    if (options.modelsDir) {
+      this.logger.warn('The use of modelsDir is deprecated. Please remove the option in the Forest initialization.');
```
Comment (rap2hpoutre) [inline src/services/config-store.js:59]: (sorry for the noise: I thought about this this morning during the standup and did not want to interrupt) Is it **deprecated** or **unsupported**? It seems it's not supported anymore, so it's not deprecated. It's removed and we have a warning.

Comment (rap2hpoutre) [inline src/services/config-store.js:59]: (sorry for the noise: I thought about this this morning during the standup and did not want to interrupt) Is it **deprecated** or **unsupported**? It seems it's not supported anymore, so it's not deprecated: it's removed and we have a warning.

Comment (jeffladiray) [reply src/services/config-store.js:59]: I think you are totally right on that. In fact, we do not `deprecate` the use of it, but we definitely remove support associated to it. Rewording is needed here and #512, so ... nice catch 👍

**Review by jeffladiray — CHANGES_REQUESTED** [review]
I tested:
- On a generated lumber project (With and without `modelsDir`)
- On our integration (Without `modelsDir`, since we're currently not using it)

And I did not see any issues.

Also, discussed with a few team member to see if they we're using `modelsDir` as an option, which does not seem to be the case. Looking at the code, I can't find any reason to keep it.

As Raphael stated, maybe you should just update the warning message to indicate that we're not supporting this option anymore instead of marking it a deprecated (IMO, it was deprecated for a long time, we're simply removing the support of it here).

Otherwise, LGTM.

**Review by jeffladiray — APPROVED** [review]
👍

**Pull request merged**